### PR TITLE
*: init docker guide

### PR DESCRIPTION
**Pull request by yichengq**
It introduces the basic way to run cfs in a docker.

Following TODO:
1. This guide doesn't support CentOS, Fedora, RHEL(solution: https://github.com/google/cadvisor/blob/master/docs/running.md)
2. Guide about running it in cluster.
3. Add cfsctl into the docker image? Good for test and play, but not good for image size.
4. Make docker run configurable
5. cadvisor in cfs should detect itself in cgroup

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM progrium/busybox
```
Comment (xiang90) [inline Dockerfile]: do we need busybox?

Comment (yichengq) [inline Dockerfile]: Yes. It provides lib needed by the cfs binary, especially cadvisor dep. I tried to build it statically, but it is harder than I think. Current image size is 18.21MB, so it is fine IMO.

Comment (yichengq) [inline Dockerfile]: Dynamic dep:

```
    linux-vdso.so.1 =>  (0x00007fff28bfe000)
    libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f94249dc000)
    libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f9424616000)
    /lib64/ld-linux-x86-64.so.2 (0x00007f9424c02000)
```

**Comment by yichengq** [comment]
Based on discussion with xiang, static building fits container-based service best because it no longer relies on external libraries. Switch to use static building now.

**Comment by missmayo** [comment]
LGTM

**Pull request merged**